### PR TITLE
DOCSP-1682 - Change default tab behavior to render inline. Added 'hid…

### PR DESCRIPTION
**Pull request by jdestefano-mongo**
…den: true' property to hide tab strips. Added 'tabs-top' directive to pull tab strip to the top of the page.

---
This change is [Reviewable](https://reviewable.io/reviews/mongodb/docs-tools/178)

### CODE REVIEW

**Comment by jdestefano-mongo** [comment]
MUST BE MERGED WITH:
https://github.com/mongodb/docs/pull/3241

File: themes/mongodb/src/js/componentTabs.js [hunk 1]
```diff
@@ -12,12 +12,12 @@ function showHideTabContent(currentAttrValue) {
...
     constructor(key) {
         this.key = key;
-        this.tabStrip = document.querySelector('.tab-strip--singleton');
+        this.tabStrips = document.querySelectorAll('.tab-strip--singleton');
 
         // Only tab sets will have a type, init and try to retrieve
         this.type = null;
-        if (this.tabStrip !== null) {
-            this.type = this.tabStrip.getAttribute('data-tab-preference');
+        if (this.tabStrips !== null) {
```
Comment (i80and) [inline themes/mongodb/src/js/componentTabs.js]: `this.tabStrips` will never be `null` given the above switch to `querySelectorAll`.

This should read `if (this.tabStrips.length > 0) {`

File: themes/mongodb/src/js/componentTabs.js [hunk 2]
```diff
@@ -61,44 +61,47 @@ class TabsSingleton {
...
+        const tabsElement = this.tabStrips[0].
+            querySelector('.tab-strip__element[aria-selected=true]');
         if (!tabsElement) { return null; }
 
         return tabsElement.getAttribute('data-tabid');
     }
 
     setup() {
-        if (!this.tabStrip) { return; }
+        if (!this.tabStrips) { return; }
```
Comment (i80and) [inline themes/mongodb/src/js/componentTabs.js]: See above

File: themes/mongodb/src/js/componentTabs.js [hunk 3]
```diff
@@ -61,44 +61,47 @@ class TabsSingleton {
...
+                };
+            }
         }
 
         this.update();
     }
 
     update() {
-        if (!this.tabStrip) { return; }
+        if (!this.tabStrips) { return; }
```
Comment (i80and) [inline themes/mongodb/src/js/componentTabs.js]: See above

**Pull request merged**